core/dive.h

```diff
@@ -169,7 +169,7 @@ struct sample                         // BASE TYPE BYTES  UNITS    RANGE
 	duration_t rbt;                   // int32_t    4  seconds  (0-34 yrs)             remaining bottom time
 	depth_t depth;                    // int32_t    4    mm     (0-2000 km)            dive depth of this sample
 	depth_t stopdepth;                // int32_t    4    mm     (0-2000 km)            depth of next deco stop
-	temperature_t temperature;        // uint32_t   4  mdegrK   (0-4 MdegK)            ambient temperature
+	temperature_t temperature;        // uint32_t   4    mK     (0-4 MK)               ambient temperature
```

Comment:
:-D

I love it

Especially in the middle of this rather dense (for me) series... A commit that I completely and fully understand...